warehouse/integrations/deltalake/deltalake.go

```diff
@@ -1168,13 +1107,6 @@ func (d *Deltalake) LoadUserTables(ctx context.Context) map[string]error {
 			columnNames(columnKeys),
 		)
 	} else {
-		if partitionQuery, err = d.partitionQuery(ctx, warehouseutils.UsersTable); err != nil {
```

Comment:
[non-blocking]
I am thinking instead of deleting this, we can add it behind a rudder-server config. Use case, dedup on 30-days window.